bleachbit/Cleaner.py

```diff
@@ -156,12 +156,9 @@ def is_running(self):
         for running in self.running:
             test = running[0]
             pathname = running[1]
-            if 'exe' == test and 'posix' == os.name:
-                if Unix.is_running(pathname):
-                    logger.debug("process '%s' is running", pathname)
-                    return True
-            elif 'exe' == test and 'nt' == os.name:
-                if Windows.is_process_running(pathname):
+            if 'exe' == test:
+                if ('posix' == os.name and Unix.is_running(pathname)) or \
+                   ('nt' == os.name and Windows.is_process_running(pathname)):
                     logger.debug("process '%s' is running", pathname)
```

Comment:
@kenrap 

Untranslated strings like this are low-hanging fruit for conversion to f-string